core/src/test/java/com/datastax/oss/driver/api/core/cql/StatementBuilderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
A unit test for a property setter on a builder is (arguably) overkill... especially since there was no existing test for other builder functionality.  That said, I had a pseudo-integration test for confirming that everything worked as expected.. so switching that over to a unit test was pretty trivial.

I'm good with scrapping this if it's judged to be unnecessary, but it shouldn't be... harmful really.